lib/std/build.zig

```diff
@@ -1808,6 +1808,29 @@ pub const LibExeObjStep = struct {
                 }
                 return;
             },
+            std.SemanticVersion => {
+                out.print(
+                    \\pub const {z}: @import("std").SemanticVersion = .{{
+                    \\    .major = {},
```

Comment:
I think we're moving towards best practice being to always provide a format specifier:

```suggestion
                    \\    .major = {d},
```